pkg/tnf/handlers/versionocp/version.go

```diff
@@ -0,0 +1,115 @@
+// Copyright (C) 2021 Red Hat, Inc.
```

Comment:
I would prefer clusterversion.go as a name of the file and package as version is very generic term.

Reply:
done